src/daemons/MetaDaemon.cpp

```diff
@@ -26,6 +27,9 @@ using nebula::Status;
 DEFINE_int32(port, 45500, "Meta daemon listening port");
 DEFINE_bool(reuse_port, true, "Whether to turn on the SO_REUSEPORT option");
 DEFINE_string(data_path, "", "Root data path");
+DEFINE_string(clusterHosts, "127.0.0.1:45500", "list of IPs split by comma,"
+                                               "all hosts in a cluster");
+DEFINE_string(clusterid_file, "data/meta.cluster.id", "File to hold the cluster id");
```

Comment:
Why not save the cluster id in kvstore on meta server.

Reply:
Yes! Now the code has changed!